Makefile

```diff
@@ -0,0 +1,28 @@
+BIN_NAME=kustomize
+
+export GO111MODULE=on
+
+all: test build
+
+test: generate-code test-lint test-go
+
+test-go:
+	go test -v ./...
+
+test-lint:
+	golangci-lint run ./...
+
+generate-code:
+	./plugin/generateBuiltins.sh $(GOPATH)
+
+build:
+	go build -o $(BIN_NAME) cmd/kustomize/main.go
+
+install:
+	cp $(BIN_NAME) $(GOPATH)/bin/$(BIN_NAME)
```

Comment:
This will have an unexpected result for users without GOPATH set or those with GOBIN set. Since we're using go modules now `go install <versionedpackagepath>` will work correctly to build/install with / without GOPATH set.

Reply:
I learnt something new today! Done